CMakeLists.txt

```
@@ -138,6 +140,20 @@ if( TRACCC_SETUP_KOKKOS )
    endif()
 endif()
 
+# Set up Alpaka.
+option( TRACCC_SETUP_ALPAKA
+   "Set up the Alpaka library" TRUE )
```

```suggestion
   "Set up the Alpaka library" ${TRACCC_BUILD_ALPAKA} )
```

Something like this would do the trick, although CMake cache variables might become incoherent if we solve it like this... I would still suggest going with this solution though.